loading.bs

```diff
@@ -440,7 +440,7 @@ following steps:
 1. Let |stream| be a [=new read buffer=] for |bodyStream|.
 1. Let |magic| be the result of [=read buffer/reading=] 8 bytes from |stream|.
 1. If |magic| is a failure, return it.
-1. If |magic| is not `` `sxg1-b1\0` ``, return a failure.
+1. If |magic| is not `` `sxg1-b2\0` ``, return a failure.
 1. Let |encodedSigLength| be the result of [=read buffer/reading=] 3 bytes from
```

Comment:
b2 includes a fallback URL here. https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl-02#section-5.3

Reply:
The UTF-8 conversion only wound up in one of the two places, so I've refactored them together.